lib/api/dump_state/Makefile

```diff
@@ -12,7 +12,7 @@ include $(CPP_SRC_HOME)/mk/defines.mk
 
 TARGET=dump_state$(EXE_EXT)
 
-ML_LIBS=$(LIB_ML_CORE) $(LIB_ML_MATHS) $(LIB_ML_MODEL) $(LIB_ML_API)
+ML_LIBS=$(LIB_ML_CORE) $(LIB_ML_MODEL) $(LIB_ML_API)
```

Comment:
I suspect this will work on macOS, but not Linux.  On Linux if the `model` library references a maths library function that has a default value that uses a static constant then the `.so` will need to be linked - something to look out for in the PR CI.